.rubocop.yml

```diff
@@ -0,0 +1,37 @@
+require: rubocop-rspec
+
+AllCops:
+  DisplayCopNames: true
+  Exclude:
+    - 'vendor/**/*'
+    - 'bin/**/*'
+    - 'db/**/*'
+    - 'app/assets/**/*'
+    - 'config/**/*'
+    - 'doc/**/*'
+    - 'spec/**/*'
+    - 'test/**/*'
+
+Rails:
+    Enabled: true
+
+Style/Encoding:
+  Enabled: true
+
+Style/StringLiterals:
+  Enabled: true
+  EnforcedStyle: single_quotes
+
+Style/Documentation:
+  Exclude:
+    - 'spec/**/*'
+    - 'app/**/*'
+
+Style/Next:
+  Enabled: false
```

Comment:
I was just wondering why it was selectively disabled, since it is a default (and default syntax convention in ruby). I would vote for having it enabled, but it's up to you all of course. @lsitu  and @VivianChu any preference? To give context, we're talking about the following:

http://rubocop.readthedocs.io/en/latest/cops_style/#stylenext